bench/build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -e
+set -x
+
+FFLAGS="-Wall -Wextra -Wimplicit-interface -fPIC -g -fcheck=all -fbacktrace"
+FFLAGS="-Wall -Wextra -Wimplicit-interface -fPIC -O3 -march=native -ffast-math -funroll-loops"
```

Comment:
adding `-march=native` isn't a fair comparison

Reply:
@vtjnash I think you put your comment on the wrong line. This is a Fortran flag, which is irrelevant for openlibm. But your comment applies to the CFLAGS below. That's a good point, I'll rerun the benchmarks to see.

Reply:
@vtjnash is `-march=native` "allowed" for openlibm?

Reply:
Sure. It's just a lot more work to run the benchmarks that way (and should be effectively a constant factor, similar to how switching processors would alter the benchmarks).

Reply:
> is `-march=native` "allowed" for openlibm?

For any project that's not targeted for a very specific use this should not be an option specified by default. It is perfectly fine as a user supplied option. It's acceptable when doing benchmarks if you make sure everything else are doing the same.

Reply:
Ok. Just to clarify what I did --- I took exactly the default options that openlibm itself provides. As I assume those are the ones that you want people to use when doing benchmarking. So I will now add `-march=native` to those as well. If there is any other option that you think I should add to make a fair comparison for the best openlibm IEEE speed, let me know.

Reply:
Anyway, I moved the `-march=native` to the CFLAGS, it provides a bit of a speedup, but not much.